extensions/amp-carousel/0.1/test/test-slides.js

```diff
@@ -382,8 +382,8 @@ describe('Slides functional', () => {
         pos: 0.6,
         min: 0,
         max: 0,
-        prevTr: prevTr,
-        nextTr: nextTr,
+        prevTr,
```

Comment:
On second thought, I think leaving the order as it is might be better. Sometimes we have similar objects of the same set of properties but different values like the following:
```javascript
object1 {
  x: x,
  y: bla,
  z: z,
}
object2 {
  x: x,
  y: bla,
  z: null,
}
```
I think converting it to
```javascript
object1 {
  x,
  y: bla,
  z,
}
object2 {
  x,
  y: bla,
  z: null,
}
```
makes more sense because object1 and object2 looks better if they have the same pattern.

@jridgewell @mkhatib @erwinmombay What do you think?